ern-container-gen/src/generators/android/AndroidGenerator.js

```diff
@@ -231,6 +237,23 @@ export default class AndroidGenerator implements ContainerGenerator {
     }
   }
 
+  populateApiImplMustacheView (pluginSourcePath: string, mustacheView: Object) {
+    const packageJson = JSON.parse(fs.readFileSync(`${pluginSourcePath}/package.json`, 'utf-8'))
+    const containerGenConfig = packageJson.ern.containerGen
+    if (containerGenConfig && containerGenConfig.apiNames) {
+      mustacheView.apiImplementaions = mustacheView.apiImplementaions ? mustacheView.apiImplementaions : []
+      for (const apiName of containerGenConfig.apiNames) {
+        let api = {}
+        api.apiName = apiName
+        api.hasConfig = containerGenConfig.hasConfig
+        api.apiVariableName = utils.camelize(apiName, true)
```

Comment:
Instead of creating an empty object and adding properties to it, just create the object directly. i.e :

```javascript
let api = {
  apiName,
  hasConfig: containerGenConfig.hasConfig,
  apiVariableName: utils.camelize(apiName, true)
}
```

You can even `push` the object directly instead of going through an intermediate `api` variable, up to you.